components/Navbar/index.js

```diff
@@ -0,0 +1,30 @@
+import {
+  NavButton,
+  NavContainer,
+  MyRecipesIcon,
+  AddRecipesIcon,
+  HomeIcon,
+} from "./Navbar-styles";
+import Link from "next/link";
+
+export default function NavigationBar() {
+  return (
+    <NavContainer>
+      <Link href="/">
+        <NavButton>
+          <HomeIcon />
+        </NavButton>
+      </Link>
```

Comment:
Links should not contain buttons - you can style links directly as:

```js
import Link from "next/link";

export const NavLink = styled(Link)`
...
`;
```

And here:

```js
<NavLink href="/">
   <HomeIcon />
</NavLink>
```